test/e2e-tests.sh

```diff
@@ -19,8 +19,14 @@ failed=0
 # Run serverless-operator specific tests.
 (( !failed )) && run_e2e_tests || failed=4
 
-# Setup serverless and run upstream e2e and conformance tests.
```

Comment:
Since we're adding a variable that controls the upgrade and e2e tests: Can we add an `if` that skips the entire section if both are false?

Reply:
I've split this into two files.